### PR TITLE
#82 Add jasmine to eslint rules

### PR DESCRIPTION
**Pull request by gson88**
Associated Issue: #82 
### Summary of Changes

- added jasmine env rule to eslint rules
- added .idea to gitignore (Webstorm / PHPStorm auto generated folder)

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -2,6 +2,9 @@
   "parser": "babel-eslint",
   "extends": ["airbnb", "prettier"],
   "plugins": ["react", "jsx-a11y", "import", "prettier"],
+  "env": {
```
Comment (ollelauribostrom) [inline .eslintrc]: Since we are using jest to run our tests, we could use the jest env instead so:
```json
"env": {
  "jest": true
}
```
That means you could also get rid of the `"jest"` global.

File: .gitignore [hunk 2]
```diff
@@ -4,4 +4,5 @@ yarn-error.log
 .DS_Store
 dist
 coverage
+.idea
```
Comment (ollelauribostrom) [inline .gitignore]: Lets put this in a separate PR 🙂

File: .eslintrc (hunk 1 above)
Comment (gson88) [reply .eslintrc]: Fixed in this pull request + added new pull request for .idea.

**Comment by ollelauribostrom** [comment]
Thanks!
